Tools/MiniBrowser/mac/SettingsController.h

```diff
@@ -27,6 +27,12 @@
 
 extern NSString * const kUserAgentChangedNotificationName;
 
+typedef NS_ENUM(NSInteger, AttachmentElementEnabledState) {
+    AttachmentElementEnabledStateDisabled = 0,
+    AttachmentElementEnabledStateEnabled,
+    AttachmentElementEnabledStateWideLayoutEnabled,
```

Comment:
@squelart Does anyone use the non-wide layout these days? Wondering if we can simplify this preference, and remove that code.

Reply:
I think that decision is such a big decision that it shouldn't affect this review?

Reply:
If it's removed, it will be then further removed from MiniBrowser too

Reply:
Yes the still-default narrow/legacy layout is still used in Mail when composing -- though eventually that will probably change.
And I believe it's used in Messages as well, but not sure.
So it's good to have the option available in minibrowser.

Reply:
Messages doesn't use `<attachment>`, but good to know about Mail!